plugin/server_audit/server_audit.c

```diff
@@ -1448,10 +1470,10 @@ static int log_proxy(const struct connection_info *cn,
                     cn->ip, cn->ip_length,
                     event->thread_id, 0, "PROXY_CONNECT");
   csize+= my_snprintf(message+csize, sizeof(message) - 1 - csize,
-    ",%.*s,`%.*s`@`%.*s`,%d", cn->db_length, cn->db,
+    ",%.*s,`%.*s`@`%.*s`,%d,%.*s", cn->db_length, cn->db,
                      cn->proxy_length, cn->proxy,
                      cn->proxy_host_length, cn->proxy_host,
-                     event->status);
+                     event->status, cn->tls_version_length, cn->tls_version);
```

Comment:
After many back-and-forths we came back to the original idea. Please, just print tls version here, without the  TLS_VERSION_KEY=